dir.json

```diff
@@ -1,18 +1,36 @@
 {
-  "A": {
-    "URI": "https://example.com/deth/v1/A/",
-    "methods": ["PUT", "DELETE"]
-  },
-  "AAAA": {
-    "URI": "https://example.com/deth/v1/AAAA/",
-    "methods": ["PUT"]
-  },
-  "SRV": {
-    "URI": "https://example.com/deth/v1/example.com/SRV/",
-    "methods": ["PUT", "DELETE"]
-  },
-  "TYPE255": {
-    "URI": "https://example.com/deth/v1/example.com/TYPE255/",
-    "methods": ["PUT", "DELETE"]
+  "resources": {
+    "http://example.com/deth/A": {
```

Comment:
These URIs should be IETF ones, and probably tag:, to avoid confusion with things that can be retrieved.